mangaki/mangaki/utils/anidb.py

```diff
@@ -241,6 +252,28 @@ def get_or_update_work(self,
 
             Staff.objects.update_or_create(work=work, role_id=nc["role_id"], artist=artist)
 
+        # Add or update tags
+        in_db_tags = Tag.objects.values_list('title', 'anidb_tag_id')
+        tags_to_add = {}
+        tags_to_update = {}
+        if not in_db_tags: # If there are no tags yet, it's safe to add
+            tags_to_add = tags
+        else: # Tags already exist so we have to prevent duplicates
+            in_db_tags_titles = [elem[0] for elem in in_db_tags]
+            in_db_tags_anidb_id = [elem[1] for elem in in_db_tags]
+            for title, tag_infos in tags.items():
+                if (not title in in_db_tags_titles and
```

Comment:
`not title in x` could be rewritten as `title not in x` (who said that Python was not nice?)